src/components/Molecules/Footer/Nav/Nav.style.js

```diff
@@ -0,0 +1,205 @@
+import styled from 'styled-components';
+
+import Link from '../../../Atoms/Link/Link';
+import hideVisually from '../../../../theme/shared/hideVisually';
+
+const NavLinkClass = styled(Link)`
+  border: 0;
+  padding: 17px 20px;
+  height: 46px;
+  width: 100%;
+  background-color: ${({ theme }) => theme.color('deep_violet')};
+  color: ${({ theme }) => theme.color('white')};
+
+  &:hover,
+  &:focus {
+    color: ${({ theme }) => theme.color('white')};
+    border-bottom: none;
+    font-weight: 500;
+  }
+`;
+
+/**
+ * Navigation menu
+ */
+const Nav = styled.nav`
+  display: block;
+  width: 100%;
+  text-align: left;
+
+  > h2 {
+    ${hideVisually};
+  }
+`;
+
+/**
+ * Sub Navigation Menu (second level)
+ */
+const SubNavMenu = styled.ul`
+  display: flex;
+  transition: max-height 0.5s ease;
+  max-height: ${({ isSubMenuOpen }) => (isSubMenuOpen ? '400px' : '0px')};
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  list-style: none outside;
+  flex-direction: column;
+
+  > li a {
+    display: ${({ isSubMenuOpen }) => (isSubMenuOpen ? 'inline' : 'none')};
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  @media ${({ theme }) => theme.breakpoint('small')} {
+    max-height: none;
+    height: auto;
+
+    &.cols--2 {
```

Comment:
Legacy?

Reply:
Nah, had to add in a little custom class to recreate the current layout from CR.com

Reply:
we could pass it as `prop` and in the styled component check if it's `2` add:
```
css`
 flex-direction: row;
 flex-wrap: wrap;
`;

Reply:
ok

Reply:
Although it what I'm doing depends on the menu content being pulled from the CMS, I don't thinking setting it as a _prop_ is the right solution here